doc/user/content/release-notes.md

```diff
@@ -48,7 +48,9 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.9.5 %}}
 
-- Timezone parsing is now case insensitive to be compatible with PostgreSQL
+- Timezone parsing is now case insensitive to be compatible with PostgreSQL.
+
```

Comment:
extra new line here

Reply:
I followed the formatting from the last few versions (line items are separated by a new line). I now realize this is not correct markdown, though (it will be interpreted as a sequence of singleton lists, so maybe we should omit the newlines as you are suggesting.